### PR TITLE
fix(spot-tv): consistent spot remote url display

### PR DESCRIPTION
**Pull request by virtuacoplenny**

### CODE REVIEW

File: spot-client/src/common/utils/window-handler.js [hunk 1]
```diff
@@ -10,7 +10,22 @@ export default {
      * @returns {string}
      */
     getBaseUrl() {
-        return window.location.origin;
+        const origin = window.location.origin;
```
Comment (paweldomas) [inline spot-client/src/common/utils/window-handler.js]: is this code changing on what the function returns ? are the JSDocs still valid here ?

Comment (virtuacoplenny) [reply spot-client/src/common/utils/window-handler.js]: Sort of. I actually think they were never really accurate to begin with. I'll tweak.

File: spot-client/src/spot-tv/ui/views/home.js [hunk 2]
```diff
@@ -336,6 +331,16 @@ export class Home extends React.Component {
...
     }
+
+    /**
+     * Generates the URL for a browser to become a Spot-Remote.
+     *
+     * @private
+     * @returns {string}
+     */
+    _getSpotRemoteConnectUrl() {
+        return `${windowHandler.getBaseUrl()}/${this.props.joinCode.toUpperCase()}`;
```
Comment (paweldomas) [inline spot-client/src/spot-tv/ui/views/home.js]: okay, now the logic for generating the join URL is duplicated in 3 places, is there a way to move this into a helper function or something ?

Comment (virtuacoplenny) [reply spot-client/src/spot-tv/ui/views/home.js]: I've moved all the logic into one component. Generally the link creation logic is for debugging and the display of the full URL was never per design. So we'll see what happens when we get more design tweaks.

Comment (virtuacoplenny) [reply spot-client/src/spot-tv/ui/views/home.js]: So it looks like in the future the base part won't even exist. But for now we'll keep it,

**Pull request merged**